action.yml

```diff
@@ -175,6 +197,16 @@ runs:
         fi
       shell: bash
 
+    - name: Run Tutor extra commands
```

Comment:
This is essentially what the step running the integration tests does:
https://github.com/eduNEXT/integration-test-in-tutor/blob/main/action.yml#L178-L188
Could we run the extra commands within that same script? If so, would it be possible to update the description and maybe the step name to indicate that it can also handle any necessary commands? This way, we could avoid adding another step and an extra input. What do you think @BryanttV

Reply:
The **Run Integration Tests** step activates the `tests_venv`, but we need the `tutor_venv` to execute Tutor commands.

Reply:
Right, I totally forgot about that